packages/cli-repl/src/cli-repl.ts

```diff
@@ -186,8 +186,17 @@ export class CliRepl implements MongoshIOProvider {
       this.bus.emit('mongosh:error', err, 'io');
     });
 
+    let jsContext = this.cliOptions.jsContext;
+    if (jsContext === 'auto' || !jsContext) {
+      jsContext = CliRepl.getFileAndEvalInfo(this.cliOptions)
```

Comment:
Do we want (didn't see where) to add the jsContext to our telemetry as a dimension to the startup event?

Reply:
@kmruiz Thanks, added it! This is probably not going to be very different from `is_interactive` in practice, but keeping track of the number of people who run it with `js_context === 'repl' && is_interactive === false` might be valuable